Ical.Net.Tests/RecurrenceTests.cs

```diff
@@ -3201,10 +3201,10 @@ public void OccurrenceMustBeCompletelyContainedWithinSearchRange()
     /// Evaluate relevancy and validity of the request.
     /// Find a solution for issue #120 or close forever
     /// </summary>
-    [Test, Ignore("No solution for issue #120 yet", Until = "2024-12-31")]
+    [Test, Ignore("No solution for issue #120 yet", Until = "2025-02-28")]
```

Comment:
As also mentioned in #685, this test is pending since 2016 and should be removed.